npc/custom/jobmaster.txt

```diff
@@ -352,91 +353,7 @@ function	Confirm_Change	{
 }
 
 function	Get_Platinum	{
```

Comment:
remove this function and call the F_GetPlatinumSkills directly in the script

Reply:
No because if remove this there was not only one line to add F GetPlatinumSkill but mamy line... so this is very efficient code line.. please check other line...

Reply:
you would only need to remove the line [ 309  function Get_Platinum; ]
and edit the line [ 327 Get_Platinum(); to callfunc "F_GetPlatinumSkills"; ] then you can remove the Get_Platinum function
the result instead of 
calling func Get_Platinum > calling func F_GetPlatinumSkills > return > return
would be 
calling func F_GetPlatinumSkills > return

Reply:
edited as your request, but i not tested it need test it

Reply:
Test before editing.

Reply:
okay i will test it